types/setting.go

```diff
@@ -117,7 +117,7 @@ var (
 		Type:        SettingTypeString,
 		Required:    true,
 		ReadOnly:    false,
-		Default:     "/var/lib/rancher/longhorn/",
+		Default:     "/var/lib/longhorn/",
```

Comment:
Don't change the path for now. There are other things associated with the path. We need to change them together.

Reply:
Dropping the change.